pkg/monitor/cluster/nodeconditions.go

```diff
@@ -32,9 +32,12 @@ func (mon *Monitor) emitNodeConditions(ctx context.Context) error {
 			}
 
 			mon.emitGauge("node.conditions", 1, map[string]string{
-				"name":   n.Name,
-				"status": string(c.Status),
-				"type":   string(c.Type),
+				// FIXME: Remove "name" and keep "nodeName" after reconfiguring
+				// monitors to use "nodeName" dimension.
+				"name":     n.Name,
+				"nodeName": n.Name,
```

Comment:
This is needed to have the ability to correlate alerts and have backwards compatibility. We have stories to fix this later